config/locales/nl.yml

```diff
@@ -66,34 +66,34 @@ nl:
   layouts:
     application:
       footer:
-        about: Over
+        about: Over Ons
         api: API
         blog: Blog
         contribute: Bijdragen
         data: Data
         designed_by: Ontwerp
         discussion_forum: Forum
         gems_served_by:
-        guides: Leren
-        help: Help
+        guides: Handleidingen
+        help: Hulp
         hosted_by: Hosting
         monitored_by: Monitoring
         optimized_by: Optimalisatie
         resolved_with: DNS
         security: Security
-        source_code: code
-        statistics: Stats
+        source_code: Code
+        statistics: Statestieken
```

Comment:
This should be "Statistieken"

Reply:
Good find, can I change this from here or would I have to create a new pull request? I'm not familiar with how this works on Github.

Reply:
Yes, it should change this in this pull-request (in order to help get it merged).

Reply:
Should be fixed now